components/feature/session/src/main/java/mozilla/components/feature/session/TrackingProtectionUseCases.kt

```diff
@@ -71,6 +73,15 @@ class TrackingProtectionUseCases(
          */
         operator fun invoke(exception: TrackingProtectionException) {
             engine.trackingProtectionExceptionStore.remove(exception)
+            // Find all tabs that need to update their tracking protection status.
+            val tabs = store.state.tabs.filter { tab ->
```

Comment:
Would we need to do that for `store.state.customTabs` too?

Reply:
Thanks for catching it, I thought we have them all mix in tabs. I will update the pr. 👍🏽

Reply:
PR update :)